IBAnimatable/GradientType.swift

```diff
@@ -6,143 +6,143 @@
 import Foundation
 
 public enum GradientType: String {
-  case SunnyDays
```

Comment:
@tbaranes I ask @TheDarkCode to leave these scripts now because we are still working on #218, and we may need to have `IBEnum` for `ColorType` and `GradientType`. Once we have a pattern for enum, we can update those scripts to use the new pattern.